app/templates/components/main-map.hbs

```diff
@@ -29,7 +29,6 @@
     hoveredFeature=map.hoveredFeature
     layerGroups=this.layerGroups
     interactivity=interactivity
-    highlightedLineFeatureLayer=highlightedLotLayer
```

Comment:
Are you sure about this? LabsMap lets you override the property: https://github.com/NYCPlanning/ember-mapbox-composer/blob/efca90484bb0a7e59ff95af640b4c180d93de62c/addon/components/labs-map.js#L95. 

Unless highlightedLotLayer is identical to what's in e-mapbox-composer.

Reply:
Hmm okay I'll check on that

Reply:
`highlightedLotLayer` highlights lots with a fill color of grey--I don't know when we would be using this, and the functionality for highlighting lots with a yellow outline comes from `ember-mapbox-composer`. So this also feels redundant, and I can't remember ever seeing another way to highlight lots in zola other than the yellow outline